src/cloud-cli/cli.ts

```diff
@@ -101,18 +114,21 @@ applicationCommands
     await getAppLogs(options.name, host, port);
   });
 
-/*** USER DATABASE MANAGEMENT ***/
+//////////////////////////////
+/* USER DATABASE MANAGEMENT */
+//////////////////////////////
+
 const userdb = program
   .command('userdb')
   .description('Manage your databases')
   .option('-h, --host <string>', 'Specify the host', DEFAULT_HOST)
-  .option('-p, --port <port>', 'Specify the port', DEFAULT_PORT)
+  .option('-p, --port <string>', 'Specify the port', DEFAULT_PORT)
 
 userdb
   .command('create')
   .argument('<string>', 'database name')
-  .option('-a, --admin <admin>', 'Specify the admin user', 'postgres')
-  .option('-W, --password <admin>', 'Specify the admin password', 'postgres')
+  .option('-a, --admin <string>', 'Specify the admin user', 'postgres')
+  .option('-W, --password <string>', 'Specify the admin password', 'postgres')
   .option('-s, --sync', 'make synchronous call', false)
```

Comment:
apparently passing <bool> to `'-s, --sync'` (`'-s, --sync' <bool>`) makes the option demand a value rather than being a flag.